### PR TITLE
Youtube mimick

### PR DESCRIPTION
**Pull request by jdmartinez1062**

### CODE REVIEW

File: Index.HTML [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+                <svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" xmlns="http://www.w3.org/2000/svg">
+                    <path d="M12 22c1.1 0 2-.9 2-2h-4c0 1.1.9 2 2 2zm6-6v-5c0-3.07-1.64-5.64-4.5-6.32V4c0-.83-.67-1.5-1.5-1.5s-1.5.67-1.5 1.5v.68C7.63 5.36 6 7.92 6 11v5l-2 2v1h16v-1l-2-2z"
+                    fill="rgb(70,70,70)">
+                    </path>
+                </svg>
+            </button>
+            <div id="avatar-div">
+                <img src="/Icons/Captura.PNG" id="user-avatar" alt="User Avatar">
+            </div>
+        </div>
```
Comment (mosaaleb) [inline Index.HTML:46]: The user avatar image is broken on live. We can fix it by removing the `/` to be `Icons/Captura.PNG`. [See this for more info](http://www.scriptingok.com/tutorial/HTML-links-2)

File: Index.HTML [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+
+            </div>
+
+
+            <div id="uploader-info">
+                <div id="upavatar-div">
+                    <img src="/Icons/Uploader avatar.PNG" id="uploader-avatar" alt="Uploader avatar">
+                </div>
+                <div id="upavatar-info-div">
```
Comment (mosaaleb) [inline Index.HTML:95]: The same concept of relative paths for this avatar.

File: Index.HTML [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+<body>
+    <nav class="nav-bar">
+        <div id="logo-div">
+            <img src="https://archive.fo/Bss88/51a3832e6af6792add876216da444db23a4bea96.png" id="logo" alt="YouTube Logo">
+        </div>
+        <form>
+            <div id="search-div">
+                <input type="text" id="search-bar" placeholder="  Search">
+            </div>
+            <button type="submit" id="search-button">
```
Comment (mosaaleb) [inline Index.HTML]: - **`[OPTIONAL]`** For simple components like search field, we can ignore the `<form>` tag. [more](https://stackoverflow.com/questions/3294572/is-input-well-formed-without-a-form)

File: Index.HTML [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+                <div id="subscribe-div">
+                    <button type="submit" id="subscribe">SUBSCRIBE</button>
+                </div>
+                
+            </div>
+        </div>
+
+        <div class="recommended-videos">
+            <div>
+                <img id="autoplay" src="Thumbnails/autoplay.png" alt="Autoplay mimick">
```
Comment (mosaaleb) [inline Index.HTML]: **`[OPTIONAL]`** We can use `<aside>` for the recommended videos for more semantic tags.

**Review by mosaaleb — CHANGES_REQUESTED** [review]
### Impressive work @mariobarrioss, @jdmartinez1062 :clap:

The UI looks very nice, yet there are some requirements needs to be met before moving on.

- [x] The project has a descriptive README.md

- [x] Files not related to the project should not be commited and added to .gitignore (e.g. `.DS_Store`)

- [x] There are some errors from your [html validator](https://validator.w3.org/). Let's fix these errors. 

**TIP** You can use html lint plugin in your preferred text editor, and it will catch these errors for you while you coding. I prefer using [html lint](https://marketplace.visualstudio.com/items?itemName=mkaufman.HTMLHint) on vscode. There is also a style lint plugins for css files.

- [x] For the page to be more alike the original page, let's add a **comments** section.

Also please see my comments below.


Please handle these requirements and submit a new code review.

#### Happy Coding :computer:

**Comment by samgaco** [comment]
### Hi @jdmartinez1062  and @mariobarrioss :hand: 

### Everything is looking good, just a few more things before we can give you the approval:

* There is still one error left showing in the [validator](https://validator.w3.org).
* You added a comments section, but a horizontal scrolling bar is showing now, it shouldn't show.
* [SUGGESTION] You could add also your contacts to the readme file and a live demo of the site using https://raw.githack.com/ .

### Please :pray: work on these points and submit back for another code review when you are done :muscle: Regards!

**Review by Ispirett — CHANGES_REQUESTED** [review]
Really Great Job with Changes @Team your project is a go :100: 
But before this can compete you need a  to resolve the stickler issues.

**Comment by samgaco** [comment]
### @jdmartinez1062 and @mariobarrioss congratulations :tada: your project is approved :clap: 

#### You can now merge to the master branch and submit the project completion form. Keep up the good work for your next projects!! :muscle:

**Pull request merged**